app/views/shared/_footer.html.erb

```diff
@@ -1,15 +1,20 @@
 <div class="main-footer">
-  <p class="text--small">
-    This site hosts a multi-benefit application and enrollment prototype, delivered by
-    Code for America and Civilla.
-  </p>
-  <p class="text--small">
-    This prototype has been designed to model how to create simpler, more user-centered
-    application experience in Michigan.
-  </p>
-  <p class="text--small">
-    This is a template that MDHHS can customize for its enrollment programs.
-  </p>
-  <div class="illustration illustration--cfa"></div>
-  <div class="illustration illustration--civilla"></div>
+  <div class="grid">
+    <div class="footer--office-links">
+      <p class="footer--office-links-header text--small">
+        MDHHS Offices
+      </p>
+      <a href="michiganbenefits.org/union" class="text--small">Union Street Office</a>
```

Comment:
let's make these dynamic --

`<%= link_to "Union Street Office", union_path, class: "text--small" %>`
`<%= link_to "Clio Road Office", clio_path, class: "text--small" %>`

Reply:
oh right I was going to do that! 🙃